src/config/README.md

```diff
@@ -0,0 +1,382 @@
+## Modules
+
+<dl>
+<dt><a href="#Products.module_OpenShift Container">OpenShift Container</a></dt>
```

Comment:
required: Appears some of the modules that have "spaces" in them are a little busted? Confirm that's intended